packages/core/src/config/minimal.ts

```diff
@@ -110,7 +110,17 @@ const minimal: PluginStyleguideConfig<'built-in'> = {
   },
   arazzoRules: {
     spec: 'error',
-    'parameters-no-body-inside-in': 'off',
+    'parameters-not-in-body': 'off',
+    'source-description-type': 'off',
+    'version-enum': 'warn',
+    'workflowId-unique': 'error',
+    'stepId-unique': 'error',
+    'sourceDescription-name-unique': 'off',
+    'workflow-dependsOn': 'off',
+    'parameters-unique': 'off',
```

Comment:
Don't we want them to be warnings?

Reply:
Maybe not. I see we have them in the recommended ruleset.

Reply:
No comments from @lornajane on those, maybe I missed smth.